yoga/model/options.py

```diff
@@ -37,6 +38,7 @@ def normalize_options(options=None):
         "no_graph_optimization",
         "no_meshes_optimization",
         "no_textures_optimization",
+        "no_fix_infacing_normals",
```

Comment:
Please update the `#Optimization flags` to `# Flags` as now we have a flag that is not an optimization